packages/apps/tools/src/components/Common/DrawerToolbar/styles.css.ts

```diff
@@ -1,66 +1,74 @@
-import { darkThemeClass, sprinkles, vars } from '@kadena/react-ui/theme';
+import { atoms, darkThemeClass, tokens } from '@kadena/react-ui/theme';
 import { style } from '@vanilla-extract/css';
 
 export const gridItemCollapsedSidebarStyle = style([
-  sprinkles({
+  atoms({
     position: 'fixed',
-    right: '$12',
-    top: '$16',
     bottom: 0,
-    backgroundColor: '$neutral1',
-    width: '$12',
     display: 'flex',
     flexDirection: 'column',
-    fontSize: '$sm',
+    fontSize: 'sm',
     zIndex: 1,
   }),
   {
+    width: tokens.kda.foundation.size.n12,
+    right: tokens.kda.foundation.size.n12,
+    top: tokens.kda.foundation.size.n16,
     height: '100vh',
-    borderLeft: `solid 1px ${vars.colors.$borderSubtle}`,
+    backgroundColor: tokens.kda.foundation.color.neutral.n1,
+    borderLeft: `solid 1px ${tokens.kda.foundation.color.border.base}`,
```

Comment:
you can use tokens.kda.foundation.border.width.hairline I think instead of 1px for consistency